### PR TITLE
update types for thumbnail images of File objects

### PR DESCRIPTION
**Pull request by StephenTangCook**
# Changes 

Updating the types of the thumbnail image dimension fields to be `number`.
Missed these in my last PR (whoops). 😅

# Ref
Snippet of example uploaded file via modal:
```
  thumb_720: '...',
  thumb_720_w: 720,
  thumb_720_h: 368,
  thumb_800: '...',
  thumb_800_w: 800,
  thumb_800_h: 409,
  thumb_960: '...',
  thumb_960_w: 960,
  thumb_960_h: 491,
  thumb_1024: '...',
  thumb_1024_w: 1024,
  thumb_1024_h: 524,
  original_w: 2654,
  original_h: 1358,
```

### CODE REVIEW

File: src_deno/request/payload/event.ts [hunk 1]
```diff
@@ -1817,11 +1817,11 @@ export interface FileItem {
   size?: number;
   source_team?: string;
   subject?: string;
-  subtype?: string;
+  subtype?: _w?: number
```
Comment (seratch) [inline src_deno/request/payload/event.ts:1820]: Can you revert the changes in this file? They look invalid.

**Review by seratch — CHANGES_REQUESTED** [review]

File: src_deno/request/payload/event.ts (hunk 1 above)
Comment (StephenTangCook) [reply src_deno/request/payload/event.ts:1820]: Yeah, was overzealous with the replace

**Review by seratch — APPROVED** [review]
